pkg/agent/agent.go

```diff
@@ -62,6 +62,13 @@ type NimbessAgent struct {
 	MetaPipelines map[string]*NimbessPipeline
 }
 
+// getPorName translates a CNI request into a Nimbess Port Identifier
```

Comment:
Will fix the doc. Oddly it does not fail golint. I wonder if that may be because it is not an export function so it does not get checked. It's creating a unique port name from information in the CNI request. Unfortunately I need to also change this to use containerID or parse out the full namespace as using just end of namespace isn't enough to be unique. Example CNI call:
K8S_POD_INFRA_CONTAINER_ID=a18ad34f25d917ad6930f0fe949401d7675aaf3393c89a1a1eeaeb606e8eae1c" ContainerID=a18ad34f25d917ad6930f0fe949401d7675aaf3393c89a1a1eeaeb606e8eae1c IfName=eth0 Netns=/proc/28662/ns/net

Ends up with port_net_eth0 which is not unique. Need to fix this.